src/redux/reducers/MissionsReducer.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable max-len */
```

Comment:
- [ ] Let's ensure we try to fix the linters errors as much as possible by avoiding disabling them and instead try to find a solution which is a good practice as a professional developer.

Reply:
fixed it shortening `"missions"` variable name into `"mis", max-len` is 100 characters per line, so I had to :)